src/redux/configureStore.js

```diff
@@ -1,12 +1,13 @@
-import { createStore, combineReducers } from 'redux';
+// import { createStore, combineReducers } from 'redux';
```

Comment:
- [ ] Kindly remove commented code from your project, you can leave comments about your code but not commented code 👍🏿 
- [ ] Please add redux-thunk to your project as a package and use as a middleware in your store, this is part of the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md#project-requirements) 👍🏿

Reply:
@prolajumokeoni I used the Redux Toolkit there is no need to use a middleware AsyncThunk is handling every action

Reply:
- [ ] Kindly skip this requested change 👍🏿